glitch-in-the-matrix/Cargo.toml

```diff
@@ -16,18 +16,15 @@ failure = "0.1"
 failure_derive = "0.1"
 http = "0.1"
 futures = "0.1"
-hyper-openssl = "0.3"
+hyper = "0.12"
```

Comment:
I removed the `compat` option of hyper, since it has been removed from hyper 0.12